bark_core/signatures/commands/approve_cmd.py

```diff
@@ -69,56 +66,44 @@ def approve(ctx, commit, gpg_key_id, ssh_key_path):
     project = ctx.obj["project"]
     repo = project.repo
 
+    key = None
+
     if not gpg_key_id and not ssh_key_path:
-        key = get_key_from_git(repo)
+        config = repo.config
+        if "gpg.format" in config and "user.signingkey" in config:
+            identifier = config["user.signingkey"]
+            type = config["gpg.format"]
```

Comment:
If `gpg.format` isn't set, does git default to "openpgp"? If yes, shouldn't this do so as well?

Reply:
It looks like it defaults to "openpgp". I just pushed a commit that resolves this.